src/ProjectService.Models.Dto/Requests/ProjectRequest.cs

```diff
@@ -14,5 +14,6 @@ public class ProjectRequest
         public Guid? DepartmentId { get; set; }
         public ProjectStatusType Status { get; set; }
         public IEnumerable<ProjectUserRequest> Users { get; set; }
+        public List<CreateProjectImageRequest> ProjectsImages { get; set; }
```

Comment:
rename request to CreateProjectRequest. And I guess better use one type of collenction here, change one of them

Reply:
done